lib/TunnelCluster.js

```diff
@@ -60,14 +64,24 @@ TunnelCluster.prototype.open = function() {
             return;
         }
 
-        debug('connecting locally to %s:%d', local_host, local_port);
+        debug('connecting locally to %s://%s:%d', local_protocol, local_host, local_port);
         remote.pause();
 
+        var allow_invalid_certificate = process.env.NODE_TLS_REJECT_UNAUTHORIZED === '0'
+        if (allow_invalid_certificate) debug('allowing invalid certificates')
+        var resolve_home = filepath => filepath.startsWith('~') ? path.join(process.env.HOME, filepath.slice(1)) : filepath;
```

Comment:
I think this is maybe the wrong place to do this?

I am guessing this issue is due to the fact that we parse `package.json` and pull the script out without it being interpreted by a shell (which would expand it) and pass it right into the node API for `localtunnel`? 

If so we should do the expansion at the point we pull it out of `package.json`

Reply:
Agreed. Moved to `storybook-chromatic`.